app/models/container_group.rb

```diff
@@ -68,6 +69,14 @@ def event_where_clause(assoc = :ems_events)
     end
   end
 
+  def tenant_identity
```

Comment:
This is fine. We can refactor this tenant_identity method later.
@gmcculloug What do you think?